content-src/asrouter/templates/Trailhead/_Trailhead.scss

```diff
@@ -264,7 +264,6 @@
 
 .trailheadCards {
   background: var(--trailhead-cards-background-color);
-  max-height: 1000px;
```

Comment:
I can't see any reason why we couldn't remove this, @rlr do you remember why this was added?